usr/local/bin/apt

```diff
@@ -106,6 +107,8 @@ elif argcommand == "deb":
 	command = sudo + " dpkg -i" + argoptions
 elif argcommand == "download":
     command = "LC_ALL=C apt-cache depends " + argoptions + " |grep -v \"Conflicts:\|Replaces:\"|awk '{print $NF}'|sed -e 's/[<>]//g'|xargs aptitude download -r"
+elif argcommand == "add-repository":
+	command = sudo + "add-apt-repository" + argoptions
```

Comment:
`command = sudo + "add-apt-repository" + argoptions`
Should be
`command = sudo + " add-apt-repository" + argoptions`

Notice space before `add-apt-repository`